packages/react/src/index.ts

```diff
@@ -45,6 +45,10 @@ export {useRefObjectAsForwardedRef} from './hooks/useRefObjectAsForwardedRef'
 export {useResizeObserver} from './hooks/useResizeObserver'
 export {useResponsiveValue} from './hooks/useResponsiveValue'
 
+// Utils
+export {createComponent} from './utils/create-component'
+export type {ComponentProps} from './utils/types/ComponentProps'
```

Comment:
Just wanted to ask, is there any reason to use this over `React.ComponentPropsWithoutRef`? (Or `React.ComponentProps`)

Reply:
Good idea! I've done this, thanks @joshblack